sigmastate/src/main/scala/org/ergoplatform/ErgoLikeContext.scala

```diff
@@ -118,17 +119,16 @@ class ErgoLikeContext(val lastBlockUtxoRoot: AvlTreeData,
       dataBoxes, boxesToSpend, newSpendingTransaction, selfIndex, extension, validationSettings, costLimit, initCost)
 
 
-  override def toSigmaContext(IR: Evaluation, isCost: Boolean, extensions: Map[Byte, AnyValue] = Map()): sigma.Context = {
-    implicit val IRForBox: Evaluation = IR
+  override def toSigmaContext(isCost: Boolean, extensions: Map[Byte, AnyValue] = Map()): sigma.Context = {
```

Comment:
If you touch old code, better add comments on the way

Reply:
added ScalaDoc in the ancestor trait